x-pack/plugins/canvas/canvas_plugin_src/renderers/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Is this a pattern we should be following, having a kind of "namespaced" collection of types, so that we can group our various types by some concept?

Reply:
Seems to be the pattern throughout Kibana, when you have a type that generically applies to a number of files.  Search for `types.ts` throughout for examples.